backend/src/packages/chaiNNer_standard/material_textures/normal_map/normalize_normal_map.py

```diff
@@ -1,8 +1,7 @@
 from __future__ import annotations
 
-import numpy as np
-
 import navi
+import numpy as np
```

Comment:
Seems like the isort rules in ruff aren't setup correctly. It seems to treat `navi` and `nodes` as third-party imports.